app/views/pages/_header.html.erb

```diff
@@ -11,13 +11,17 @@
         <li class="nav-item">
             <%= link_to "Sign Out", destroy_employee_session_path, method: :delete, class:"nav-link" %>
             </li>
+             <li class="nav-item">
+              <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Available kudos:<%= current_employee.number_of_available_kudos %></a>
```

Comment:
Maybe instead of using `a` witch creates link, you can use `span` to add styling to text
https://developer.mozilla.org/pl/docs/Web/HTML/Element/span